docs/RFCS/20220120_row_level_ttl.md

```diff
@@ -29,7 +29,7 @@ CREATE TABLE tbl (
    text TEXT,
    expiration TIMESTAMPTZ,
    should_delete BOOL,
-) WITH (ttl = 'on', ttl_expiration_expression = 'if(should_delete, expiration, NULL)');
+) WITH (ttl_expiration_expression = 'if(should_delete, expiration, NULL)');
```

Comment:
can you add a section near the top here with something like

```
# Changelog

* 2022-07-07: Removed the `ttl_automatic_column` <brief summary about why> <link to this PR>

```

for a paper trail?